server/plugin/command_test.go

```diff
@@ -802,6 +847,57 @@ func TestHandleEditSubscription(t *testing.T) {
 	}
 }
 
+func TestHandleCreateIncident(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	args := &model.CommandArgs{
+		UserId: testutils.GetID(),
+	}
+
+	t.Run("HandleCreateIncident: Success", func(t *testing.T) {
+		defer mockAPI.AssertExpectations(t)
+		assert := assert.New(t)
+		mockAPI.On("PublishWebSocketEvent", mock.AnythingOfType("string"), mock.Anything, mock.AnythingOfType("*model.WebsocketBroadcast")).Return()
+		p.SetAPI(mockAPI)
+		resp := p.handleCreateIncident(args)
+		assert.EqualValues("", resp)
+	})
+}
+
+func TestHandleCreateRequest(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	args := &model.CommandArgs{
+		UserId: testutils.GetID(),
+	}
+
+	t.Run("HandleCreateRequest: Success", func(t *testing.T) {
+		defer mockAPI.AssertExpectations(t)
+		assert := assert.New(t)
+		mockAPI.On("PublishWebSocketEvent", mock.AnythingOfType("string"), mock.Anything, mock.AnythingOfType("*model.WebsocketBroadcast")).Return()
+		p.SetAPI(mockAPI)
+		resp := p.handleCreateRequest(args)
+		assert.EqualValues("", resp)
```

Comment:
here resp is going to be empty everytime or just for the successful and non empty for error case ?

Reply:
yes it is going to be empty every time

Reply:
then why are we checking this anyways if it going to be empty everytime ?

Reply:
discussed